hasura/graphql-engine
# Enable add/modify/delete for event triggers and remote schemas in migration mode, fix #1875

### Description
User will now see controls for add/modify/delete event triggers and remote schemas when in migration mode. 

### Affected components 

- Console

### Related Issues
close #1875

### Solution and Design
Removed check for migrationmode when not needed

### Steps to test and verify
set migration mode on, 
visit the events trigger page, add  a simple event trigger, modify it, then delete it
visit the remote schemas page, use the Glitch example to make a remote schema,
modify it, and delete it.
visit the Data page and see that you can't modify the DDL

### Limitations, known bugs & workarounds
You can still trick the system by entering SQL
